pkg/pillar/cmd/msrv/handlers.go

```diff
@@ -709,3 +709,76 @@ func (msrv *Msrv) withPatchEnvelopesByIP() func(http.Handler) http.Handler {
 		})
 	}
 }
+
+// handleActivateCredentialGet handles get request of the activate-credential exchange,
+// it returns a json containing the EK, AIK public keys and AIK name.
+func (msrv *Msrv) handleActivateCredentialGet() func(http.ResponseWriter, *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		if r.Method == http.MethodGet {
+			ekPubByte, aikPubByte, aiKnameMarshaled, err := getActivateCredentialParams()
+			if err != nil {
+				msrv.Log.Errorf("handleActivateCredntialGet: %v", err)
```

Comment:
I suggest a search and replace of "Credntial"

Reply:
fixed.